G2H.Portal.Web/Views/Bases/ImageBase.razor.cs

```diff
@@ -0,0 +1,28 @@
+﻿// --------------------------------------------------------------------------------
+// Copyright (c) Christo du Toit. All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// FREE TO USE TO HELP SHARE THE GOSPEL
+// Mark 16:15 NIV "Go into all the world and preach the gospel to all creation."
+// https://mark.bible/mark-16-15 
+// --------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace G2H.Portal.Web.Views.Bases
+{
+    public partial class ImageBase : ComponentBase
+    {
+        [Parameter]
+        public string Src { get; set; }
```

Comment:
@cjdutoit properties must be fully named. Such as `Source`, `AlternativeText` and others.

Reply:
@hassanhabib , thanks I have submitted a code rub for this #7